meta-facebook/yv3-vf/src/platform/plat_isr.c

```diff
@@ -127,6 +132,15 @@ static void m2_presen_evt(uint32_t dev, uint32_t status)
 void prsnt_int_handler(uint32_t idx, uint32_t arg1)
 {
 	const uint8_t is_prsnt = m2_prsnt(idx);
+	uint8_t retry = 3;
+
+	while (retry--) {
```

Comment:
Please change the while loop for retries to a 'for' loop.  It is safer to use a for loop than a while loop for these types of operations plus it makes code maintenance easier. Thanks!

Reply:
Hi @garnermic,
fix in new commit(b39869d20ca4143b16a7bcddb41cb4b3f1c211ad)
Thanks!